cosmwasm/utils.js

```diff
@@ -737,6 +762,15 @@ const encodeParameterChangeProposal = (options) => {
     };
 };
 
+const encodeMigrateContractProposal = (config, options, chainName) => {
+    const proposal = MigrateContractProposal.fromPartial(getMigrateContractParams(config, options, chainName));
+
+    return {
+        typeUrl: '/cosmwasm.wasm.v1.MigrateContractProposal',
+        value: Uint8Array.from(MigrateContractProposal.encode(proposal).finish()),
+    };
+};
+
 const encodeSubmitProposal = (content, config, options, proposer) => {
```

Comment:
We should create a test-cosmwasm gh action similar to test-sui/test-evm that runs cmds against a local node. Can you create a ticket and take that on in the future?